cocotb/__init__.py

```diff
@@ -39,7 +39,10 @@
 
 import cocotb.handle
 from cocotb.scheduler import Scheduler
-from cocotb.log import SimBaseLog, SimLog
+try:
+    from cocotb_custom.log import SimBaseLog, SimLog
+except ModuleNotFoundError:
+    from cocotb.log import SimBaseLog, SimLog
```

Comment:
Tries loading user code from a known location. Requires no more environment variables. User required to add the correct path to their PYTHONPATH. The `cocotb_custom` namespace can be reserved for future use in the project for user customization points. This parallels the consensus of `cocotb_ext` or `cocotbext` for the naming of the cocotb extension module namespace.